server/app/views/questiontypes/FileUploadQuestionRenderer.java

```diff
@@ -73,6 +73,7 @@ protected DivTag renderInputTag(
             label()
```

Comment:
question (potentially blocking): What do you think about making this a button? I noticed when I was tab navigating I can navigate to "Choose File" now, but I can't interact with it (I expected that hitting "enter" would allow me to upload a file). I think changing this to a button might resolve that, but I'm not sure.

Reply:
Actually, I take it back, you can interact with it with "control option space" which is how you interact with buttons! So I think the functionality is fine, but I'm still curious what you think of making it a button. Right now the screen reader tells me I'm on a "group" which isn't super clear.

Reply:
Ah I gotcha. I'll spend a little time futzing with it, but seems like it might not be possible.

Reply:
Awesome thanks! Eager to hear if you find anything :)

Reply:
I think this should work!
`.attr("role", "button")`

When I tried it out locally with my screen reader it recognized it as a button and prompted me to interact with it. Here is an [SO post](https://stackoverflow.com/questions/55197141/how-to-make-screenreader-not-recognise-an-element-as-an-element-group) that helped me understand the role of "role" if that's helpful!

Reply:
Ah, just the "role" alone didn't work for me locally, so I got rid of that. But having both "role" and "tabindex" works and I agree that this label should have the button role!

Reply:
Updated to put the role on a `div` element, since https://www.w3.org/TR/html-aria/  indicates `label`s aren't allowed to have roles. (Thanks for finding that doc Rocky!)